src/questions/stable_coins.yaml

```diff
@@ -1,22 +1,960 @@
+  hint: NoHint
+  explanation: BUSD is a fiat-backed and FRAX is crypto-backed stablecoin.
+  answerKeys:
+    - A
+    - C
+  subTopics:
+    - commodity-backed
+  difficultyLevel: Low
+  choices:
+    - content: PAXG
+      key: A
+    - content: BUSD
+      key: B
+    - content: XAUT
+      key: C
+    - content: FRAX
+      key: D
+- uuid: 9ee8554b-d3ed-477a-8ea4-2de47e2f029f
+  type: SingleChoice
+  content: The value of one Paxos Gold (PAXG) is pegged to the value of which of the following?
+  hint: NoHint
+  explanation: The value of one PAXG is pegged 1:1 to the value of one fine troy ounce of a London Good Delivery gold bar.
+  answerKeys:
+    - A
+  subTopics:
+    - commodity-backed
+  difficultyLevel: Medium
+  choices:
+    - content: One Troy Ounce of a London Good Delivery gold bar
+      key: A
+    - content: One Troy Pound of a London Good Delivery gold bar
+      key: B
+    - content: One Ounce of a London Good Delivery gold bar
+      key: C
+    - content: One Pound of a London Good Delivery gold bar
+      key: D
+- uuid: 8152931c-e79f-422f-80c3-6259c878257f
+  type: SingleChoice
+  content: What is the ratio in which one Tether Gold (XAUT) is pegged?
+  hint: equal
+  explanation: Like Paxos Gold, one XAUt token is pegged 1:1 to the value of one fine troy ounce of gold on a London Good Delivery bar.
+  answerKeys:
+    - D
+  subTopics:
+    - commodity-backed
+  difficultyLevel: Low
+  choices:
+    - content: "1:100"
+      key: A
+    - content: "1:10"
+      key: B
+    - content: "10:1"
+      key: C
+    - content: "1:1"
+      key: D
+- uuid: aaaeed74-6470-4f57-b28a-80fec5135d29
+  type: SingleChoice
+  content: Which of the following is used in Algorithmic Stablecoins?
+  hint: NoHint
+  explanation: On-chain collateral or conventional off-chain collateral are not used in this approach. Instead, the value of a second token serves as collateral for the stablecoin.
+  answerKeys:
+    - C
+  subTopics:
+    - algorithmic
+  difficultyLevel: Low
+  choices:
+    - content: On-chain collateral
+      key: A
+    - content: Off-chain collateral
+      key: B
+    - content: A Second token
+      key: C
+    - content: No collateral is used
+      key: D
+- uuid: e272817a-a463-468c-a692-8bf9d1d66e13
+  type: SingleChoice
+  content: UST is an example of Algorithmic stablecoin. True or False?
+  hint: NoHint
+  explanation: UST is an example of Algorithmic stablecoin.
+  answerKeys:
+    - A
+  subTopics:
+    - algorithmic
+  difficultyLevel: Low
+  choices:
+    - content: True
+      key: A
+    - content: False
+      key: B
+- uuid: e8a9fcfd-5250-4d7b-8314-3e5e480fdf63
+  type: MultipleChoice
+  content: Choose the correct statement(s).
+  hint: NoHint
+  explanation: Since there is no other asset backing these stablecoins, if the price drops below the targeted level, an algorithm will sell tokens, and if the price rises above the acceptable level, it will supply tokens.
```

Comment:
This is a very important question and the language needs to be rephrased. Its explained very nicely here

https://www.gemini.com/cryptopedia/what-are-stablecoins-how-do-they-work
https://decrypt.co/resources/what-are-algorithmic-stablecoins